content/blog/coredns-in-kubernetes-via-kubeadm.md

```diff
@@ -151,6 +164,7 @@ The upgrade with CoreDNS as the default DNS can then be performed using `kubeadm
 ~~~
 
 To verify CoreDNS is running, we can check the pod status, deployment and the configmap in the node.
```

Comment:
I think you should add a title here 'Verifying CoreDNS is providing the service' (or something similar). I guess that will be for whatever process of install-or-upgrade.